src/strings/en-us.json

```diff
@@ -0,0 +1,2580 @@
+    "File-Blame": "File Blame",
+    "File-CanOnlyEditText": "Only text files can be edited",
+    "File-CouldNotOpenInEditor": "Could not open file in editor",
+    "File-CouldNotCreate": "Could not create file",
+    "File-CouldNotCreateOutOfRepo": "Cannot create a file outside of the repository!",
+    "File-CouldNotDelete": "Could not delete file",
+    "File-Create": "Create File",
+    "File-Created": "Created '{0}'",
+    "File-Delete": "Delete file",
+    "File-DeleteInWorkDir": "Delete file in working directory",
+    "File-Deleted": "Deleted '{0}'",
+    "File-DoesNotExist": "File does not exist",
+    "File-ConfirmDeletePath": "Are you sure you want to delete '{0}'?",
+    "File-Open": "Open file",
+    "File-Edit": "Edit file",
+    "File-OpenInDefaultProgram": "Open file in default program",
+    "File-OpenInDefaultEditor": "Open in external editor",
+    "File-OpenInNamedEditor": "Open in {0}",
+    "File-OpenWithName": "Open {0}",
+    "File-Reset": "Discard changes",
+    "File-ShowFolderContents": "Open folder",
+    "File-ShowFolderContentsMac": "Open in Finder",
+    "File-ShowInFolder": "Show in folder",
+    "File-ShowInFolderMac": "Show in Finder",
+    "File-ShowInFolderMacWithName": "Show {0} in Finder",
+    "File-ShowInFolderWithName": "Show {0} in folder",
+    "File-StatusAdded": "ADDED",
+    "File-StatusDeleted": "DELETED",
+    "File-StatusRenamed": "RENAMED",
+    "File-StatusRenamedFrom": "from",
+    "File-StatusRenamedTo": "to",
+    "File-UnstageAndRemove": "Unstage and delete file",
+
+    "Folder-NewFile": "Create a file in this folder",
+    "Folder-MarkConflictedFilesAsResolved": "Mark all resolved",
+    "Folder-Open": "Open folder",
+    "Folder-ResetAll": "Reset folder",
+    "Folder-StageAll": "Stage folder",
+    "Folder-UnstageAll": "Unstage folder",
+    "Folder-DiscardAllChanges": "Discard all changes in folder",
+
+    "FileHistory": "File History",
+    "FileHistory-AuthorInfo": "Authored by {0} on {1}",
+    "FileHistory-CannotLinkToGraph": "This commit is not visible on the graph",
+    "FileHistory-CommitShaCopied": "Commit Sha copied!",
+    "FileHistory-DiffView": "Diff View",
+    "FileHistory-End": "End of History",
+    "FileHistory-FileView": "File View",
+    "FileHistory-LinkToGraph": "View {0} on graph",
+    "FileHistory-BlameButtonLabel": "Blame",
+    "FileHistory-ShowBlame": "Show blame details",
+    "FileHistory-UnknownAuthorName": "Unknown",
+
+    "FuzzyFinder-DefaultPlaceHolder": "Search for commands and actions (e.g., Open Repo)",
+    "FuzzyFinder-BlamePlaceholder": "Search for a file to view who did what (i.e. its blame)",
+    "FuzzyFinder-CheckoutPlaceholder": "Search for a branch to checkout",
+    "FuzzyFinder-CloseTab": "Close Tab",
+    "FuzzyFinder-CreateAnnotatedTag": "Create Annotated Tag",
+    "FuzzyFinder-CreateBranch": "Create Branch",
+    "FuzzyFinder-CreateFilePlaceholder": "Enter a new file name",
+    "FuzzyFinder-CreateTag": "Create Tag",
```

Comment:
Comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment 20

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment 30

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment 40

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment 50

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment 60

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment 70

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment 80

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment 90

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
comment

Reply:
![pepeDancegif](https://user-images.githubusercontent.com/20191027/114932878-e2550f00-9dec-11eb-856c-d142cc087231.gif)

The 100th comment in this thread

Reply:
a me-me as the kids would say https://i.ytimg.com/vi/F9C0T_O7cIo/maxresdefault.jpg